### PR TITLE
ozaudit [L05] Interfaces are missing functions present in their imple…

### PR DESCRIPTION
**Pull request by ewilz**
…mentations

### Description

Include all public/external implemented functions in respective interfaces

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

Tested. needed to remove Interface extensions from certain mock contracts since they don't implement the entire interface.
### Related issues

https://github.com/celo-org/celo-labs/issues/589
### Backwards compatibility

only interface changes

### CODE REVIEW

File: packages/protocol/contracts/common/interfaces/IMetaTransactionWallet.sol [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
   function executeTransaction(address, uint256, bytes calldata) external returns (bytes memory);
   function executeTransactions(
     address[] calldata,
     uint256[] calldata,
     bytes calldata,
     uint256[] calldata
   ) external;
-  function executeMetaTransaction(address, uint256, bytes calldata, uint8, bytes32, bytes32)
+
+  // view functions
```
Comment (yorhodes) [inline packages/protocol/contracts/common/interfaces/IMetaTransactionWallet.sol:16]: love the grouping

File: packages/protocol/contracts/identity/interfaces/IAttestations.sol [hunk 2]
```diff
@@ -1,24 +1,22 @@
 pragma solidity ^0.5.3;
 
 interface IAttestations {
-  function setAttestationRequestFee(address, uint256) external;
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256);
```
Comment (yorhodes) [inline packages/protocol/contracts/identity/interfaces/IAttestations.sol]: don't duplicate across interfaces (`ICeloVersionedContract`) as discussed

File: packages/protocol/contracts/common/interfaces/IMetaTransactionWallet.sol [hunk 3]
```diff
@@ -1,16 +1,34 @@
 pragma solidity ^0.5.3;
 
 interface IMetaTransactionWallet {
-  function setSigner(address) external;
+  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256);
```
Comment (yorhodes) [inline packages/protocol/contracts/common/interfaces/IMetaTransactionWallet.sol]: ditto

**Review by yorhodes — APPROVED** [review]
approved pending the deduping

**Comment by ewilz** [comment]
@yorhodes confused about the circleci failures? something about dependency vulnerability?

**Pull request merged**